### DISCUSSION TITLE
Why is Homebrew the recommended Python installation for MacOS?

### DISCUSSION THREAD

**Discussion by tom-new** [firedrakeproject/firedrake#3102] [Firedrake support] [answered]
Why is Homebrew the recommended Python installation for MacOS? Perhaps I am doing something stupid, but every time Homebrew decides to update Python, my Firedrake installation breaks, and I have to reinstall it (which can take a while). This seems pretty inconvenient.

**Comment by dham** [comment, marked as answer]
Hi Tom. Homebrew is recommended because it provides several of our dependencies in one place. We also have repeated bad experiences with Anaconda, and the Python that ships with MacOS is known to be broken. I agree that Homebrew's habit of randomly updating is somewhat irritating. I think you could, for example, use pyenv if Homebrew vexes you too much.
